CentralManagement/ChocoCCM Command Reference/Set-CCMNotificationStatus.md

```diff
@@ -0,0 +1,82 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Set-CCMNotificationStatus
+
+## SYNOPSIS
+Turn notifications on or off in CCM

```

Comment:
I'd be inclined to just have two commands for this; `Enable-CcmNotification` and `Disable-CcmNotification`. Simpler names, clearer purpose. They can call into the same internal command.